client/state/plugins/wporg/test/test-actions.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';;
```

Comment:
some are against semicolons altogether in JavaScript, maybe @johnHackworth is just fighting back - "_**two**_ semicolons to make up for all those missing ones!"